Dockerfile

```diff
@@ -29,8 +29,13 @@ RUN curl --fail --location --remote-name https://github.com/pmem/ndctl/archive/v
 RUN tar zxvf v${NDCTL_VERSION}.tar.gz && mv ndctl-${NDCTL_VERSION} ndctl
 WORKDIR /ndctl
 RUN ./autogen.sh
-RUN ./configure ${NDCTL_CONFIGFLAGS}
-RUN make install
+# We install into /usr/local (keeps content separate from OS) but
+# then symlink the .pc files to ensure that they are found without
+# having to set PKG_CONFIG_PATH. The .pc file doesn't contain an -rpath
+# and thus linked binaries do not find the shared libs unless we
+# also symlink those.
+RUN ./configure --prefix=/usr/local ${NDCTL_CONFIGFLAGS}
+RUN make install && ln -s /usr/local/lib/pkgconfig/libndctl.pc /usr/lib64/pkgconfig/ && ln -s /usr/local/lib/pkgconfig/libdaxctl.pc /usr/lib64/pkgconfig/ && for i in /usr/local/lib/lib*.so.*; do ln -s $i /usr/lib64; done
```

Comment:
@pohly Would you mind splitting this long line.

Reply:
Done.